etcdserver/etcdserverpb/rpc.proto

```diff
@@ -510,6 +510,9 @@ message Compare {
     // value is the value of the given key, in bytes.
     bytes value = 7;
   }
+  // range_end compares over the range [key, range_end). See RangeRequest.
+  bytes range_end = 8;
```

Comment:
but i can see people might want multiple targets eventually. but it is something easy to support i feel.